web/themes/joinup/templates/parts/rdf-entity--collection--listing-item--add-collection.html.twig

```diff
@@ -4,11 +4,5 @@
  * Themeing for rdf entities.
  */
 #}
-{%
-  set classes = [
-    'mdl-cell--middle'
-  ]
-%}
-<div class="{{ classes|join(' ') }}">
-  {{ add_collection_link }}
-</div>
+ 
+{{ add_collection_link }}
```

Comment:
This separated tile is not needed, so I removed it. I also changed structure for create collection tile and created new component for it.